src/h5web/explorer/Explorer.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { fireEvent, screen } from '@testing-library/react';
+import { DOMAIN, renderApp } from '../test-utils';
+
+describe('Explorer', () => {
+  test('select root group by default', async () => {
+    renderApp();
```

Comment:
Is the app created without any mock data here ?

Reply:
Nothing's changed, it renders the `App` with the `MockProvider`, so it renders exactly the same data as the `/mock` route.

Reply:
Read until the end: it has mock data.

Then, perhaps the check on `Nothing to visualize` is a bit tricky as the root node might have some visualisations with NeXuS.

Reply:
Hmm good point... Seems to be passing when it shouldn't...